lib/gooddata/lcm/lcm2.rb

```diff
@@ -131,7 +131,6 @@ def respond_to_missing?(name, *_args)
         EnsureTechnicalUsersDomain,
         EnsureTechnicalUsersProject,
         SynchronizeLdm,
-        ApplyCustomMaql,
```

Comment:
Is this really a part of this change?

Reply:
Yes. This action has never worked and tests of conflicting ldm would fail with it.